charts/kong-collectorapi/values.yaml

```diff
@@ -0,0 +1,49 @@
+image:
+  # repository: kong-docker-kong-brain-immunity-base.bintray.io/kong-brain-immunity
+  # tag: latest
+  pullPolicy: IfNotPresent
+  repository: kong-docker-kong-brain-internal-builds.bintray.io/kong-brain
+  tag: master
+
+imagePullSecrets:
+  # - name: bintray-kong-brain-immunity
+  - name: bintray-kong-brain-internal
+
+waitImage:
+  repository: busybox
+  tag: latest
+  pullPolicy: IfNotPresent
+
+
+# Set this to the Kong Admin API hostname
+kongAdminHost: my-kong-kong-admin
+# Set this to the Kong Admin API port
+kongAdminServicePort: 8001
+# Set this to the Kong Admin API protocol
+kongAdminProtocol: http
+containerPort: 5000
+# Set a nodePort which is available
+nodePort: 31555
+service: 
+  type: NodePort
+  port: 5000
+
+# Enable this for running integration tests
+testendpoints: 
+  enabled: false
```

Comment:
nit: Since we don't expect users to use this, we should move this to the end of the file.